### DISCUSSION TITLE
taro4.0.4 h5打包报错

### DISCUSSION THREAD

**Discussion by yamlling** [NervJS/taro#16316] [Q & A]
taro4.0.4 h5打包报错 有谁遇到过或者有解决方案吗?
使用taro-cli空模板打包也是失败的
![1723706957709](https://github.com/user-attachments/assets/7a4dd009-757a-4510-a3d3-003d2b4cec25)

**Comment by yamlling** [comment]
技术栈 react+vite

**Comment by qq1037305420** [comment]
windows路径问题 用linux或者改下源码 normalize 路径一下就行了
